gradle.properties

```diff
@@ -1,3 +1,3 @@
 vcs_url=https://github.com/th2-net/th2-read-db
 release_version=0.0.0
-version_suffix=
\ No newline at end of file
+version_suffix=
```

Comment:
issue (non-blocking): Please, revert this change. No need to change the file in PR if we don't need those changes